src/solver/type_getters.jl

```diff
@@ -511,6 +500,18 @@ function get_callbacks(parsed_args, sim_info, atmos, params, comms_ctx)
     end
 
     dt_cf = FT(time_to_seconds(parsed_args["dt_cloud_fraction"]))
+
+    if parsed_args["log_progress"] && !sim_info.restart
+        @info "Progress logging enabled."
+        callbacks = (
+            callbacks...,
+            call_every_n_steps(
+                (integrator) -> print_walltime_estimate(integrator);
+                skip_first = true,
+            ),
+        )
+    end
+
```

Comment:
Can we make `cloud_fraction_model_callback` below optional too?

Reply:
Ah, I'll follow up with that later.

Reply:
I added a check for the `dt`. When negative, the callback is not added.